colo/docker/uploader.sh

```diff
@@ -17,7 +17,7 @@ FULL_S3_ACCESS_POINT_PATH=${S3_ACCESS_POINT}/${FILENAME}
 FULL_S3_URL_PATH=${S3_URL}/${FILENAME}
 
 # Send to AWS
-aws s3 cp ${FILEPATH} ${FULL_S3_ACCESS_POINT_PATH}
+aws s3 cp --quiet ${FILEPATH} ${FULL_S3_ACCESS_POINT_PATH}
```

Comment:
```suggestion
aws s3 cp --quiet ${FILEPATH} ${FULL_S3_ACCESS_POINT_PATH} || exit 1
```

This way the SQS message below is only sent if the copy was successful.